tools/tolua/genbindings.py

```diff
@@ -37,6 +37,52 @@ def _check_python_bin_env():
 
     return PYTHON_BIN
 
+def _find_first_file_in_dir(dir, fn):
+    if os.path.isfile(dir):
+        if os.path.basename(dir) == fn:
+            return os.path.join(os.path.dirname(dir), fn)
+        else :
+            return None
+    elif os.path.isdir(dir):
+      for subdir in os.listdir(dir):
+          searchPath = _find_first_file_in_dir(os.path.join(dir, subdir), fn)
+          if searchPath is not None:
+              return searchPath
```

Comment:
what happen if searchPath is None?

Reply:
the function will return None by default behavior